app/models/mets_document/mods_document.rb

```diff
@@ -20,7 +20,7 @@ def initialize(element)
 
     def title
```

Comment:
This is good, but we need a `sort_title` field too which is the title without the nonSort bit.

Reply:
I will add this as well. Thanks!